### PR TITLE
Mobile nav

### PR DESCRIPTION
**Pull request by denscholar**
### Changes made:
Updated CSS and HTML files. This feature had been worked on and approved before now, I had to make some changes I feel are necessary to the feature.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "liveServer.settings.port": 5501,
+  "liveServer.settings.port": 5502,
   "livePreview.defaultPreviewPath": "/index.html"
```
Comment (MrBrN197) [inline .vscode/settings.json:3]: - [ ] [OPTIONAL] you can add this .vscode file to your .gitingore. usually personal setting files that don't have to do with the project can be added to the .gitignore file.

File: index.js [hunk 2]
```diff
@@ -12,9 +12,9 @@ const email = document.querySelector('.email');
...
 const x = element.querySelector('small');
 const form = document.querySelector('.form-input');
-const [fullName, firstName, lastName] = form.elements;
-
+const [fullName, firstName, lastName, message] = form.elements;
 const mediaqueryList = window.matchMedia('(max-width: 768px)');
+const formData = document.querySelector('.form-input'); // gets the form element
 
 const screenTest = (e) => {
   if (e.matches) {
```
Comment (MrBrN197) [inline index.js:20]: - [ ] Your javascript functionality works very well. I'd like to point out that your hamburger design doesn't match the Figma design. we want you to match the design as close as possible. you can have the design look like that when you hover or active over the hamburger button. button while it is stationary please leave the design as per the Figma.
## Your Design
image

## Figma Design
image

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @denscholar 👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔ Professional README
- ✔ JavaScript functionality works as expected

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
## Hi @denscholar, 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved emoji



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**